### PR TITLE
Added wok recipe

### PR DESCRIPTION
**Pull request by HenrikJantti**
This is the wok recipe i use for when i make it

### CODE REVIEW

**Review by egavazzi — CHANGES_REQUESTED** [review]
How do you boil your water?

File: henrikwok.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Ingredients:
+* Your favorite noodles
+* Wok mix of your choice
+* Some source of protein (Chicken/Pork in strips is my go to)
+* Oyster sauce
+* Soy sauce
+* Any decired spices
+
+Instruction:
+*start boiling your water
```
Comment (HenrikJantti) [inline henrikwok.txt:10]:
```suggestion
*start boiling your water by adding water to a pot and putting it on the stove
```

File: henrikwok.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+*Start frying you protein
+*At this point the water sould be boiling add your noodles
+*when protein is done add wok mix
```
Comment (bast) [inline henrikwok.txt:13]:
```suggestion
Instruction:
*start boiling your water
*Start frying you protein
*At this point the water should be boiling add your noodles
*when protein is done add wok mix
```

Comment (Hegebf) [reply henrikwok.txt]:
```suggestion
Instruction:
*start boiling your water
*Start frying your protein
*At this point the water sould be boiling add your noodles
*when protein is done add wok mix
```

File: henrikwok.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Ingredients:
+* Your favorite noodles
+* Wok mix of your choice
+* Some source of protein (Chicken/Pork in strips is my go to)
+* Oyster sauce
+* Soy sauce
+* Any decired spices
```
Comment (egavazzi) [inline henrikwok.txt:7]:
```suggestion
* Any desired spices
```

File: henrikwok.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Ingredients:
```
Comment (makorzen) [inline henrikwok.txt:1]: how about some beer to wash it down?

File: henrikwok.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+*start boiling your water
+*Start frying you protein
+*At this point the water should be boiling add your noodles
+*when protein is done add wok mix
+*Now add as mutch oyster sauce and soy sauce as you deem necessary (large amounts)
+*And spices
+*when the noodles are done add them to the pan (Whithout the water)
+*At this point you can do some washing up (if you want)
+*When you feel that you have had the noodles in the pan for long enough add them tou your favorit food serving device.
+* Eat your wok with prefered utensils
```
Comment (makorzen) [inline henrikwok.txt:19]:
```suggestion
* Eat your wok with prefered utensils
* do the dishes
```

File: henrikwok.txt [hunk 6]
```diff
@@ -0,0 +1,19 @@
+Ingredients:
+* Your favorite noodles
+* Wok mix of your choice
+* Some source of protein (Chicken/Pork in strips is my go to)
+* Oyster sauce
+* Soy sauce
+* Any decired spices
+
```
Comment (HenrikJantti) [inline henrikwok.txt:8]:
```suggestion
* Beer
```

File: henrikwok.txt (hunk 5 above)
Comment (HenrikJantti) [reply henrikwok.txt:19]: This was a good idea

File: henrikwok.txt (hunk 4 above)
Comment (HenrikJantti) [reply henrikwok.txt:1]: This is now done

**Review by egavazzi — APPROVED** [review]

**Pull request merged**